common/src/utils/passports/genMockIdDoc.ts

```diff
@@ -131,6 +131,7 @@ function sign(
         const msgHash = hash(hashAlgorithm, eContent, 'hex');
 
         const signature = keyPair.sign(msgHash, 'hex');
+        // @ts-ignore-error THIS SEEMS LIKE A LEGIT PROBLEM -- aaronmgdr
         const signatureBytes = Array.from(Buffer.from(signature.toDER(), 'hex'));
```

Comment:
the types here are not compatible. not sure if we just ignore because we know something it doesnt since that will same as it is now